### ISSUE TITLE
Improve the readability of the documentation for the EDR query patterns

### ISSUE THREAD

**Issue by m-burgoyne** [opengeospatial/ogcapi-environmental-data-retrieval#319]
Due to the layout and length of text descriptions, the current documentation can be hard to comprehend.  The presentation of the information could be simplified by moving to a table based summary which provided links to detailed information. 

for example the Position query information could be presented as follows:

|Query Parameter| Type | Required|Description
|-------|------|-----|------|
| [**coords**](position_coords) |WKT string|**Yes**| The coordinates are defined by a Well Known Text (WKT) string
| z     |String  |No|  The vertical level to return data for (available options listed in the extent section of the [collections response](collection_metadata_desc) response)
| [datetime](datetime)  |String  |No| Datetime range to return data (available range listed in the the extent section of the [collections response](collection_metadata_desc) response)  |
|[parameter-name](parameter-name) |String  |No| Comma delimited list of parameter names available options listed in the  [collections response](collection_metadata_desc)
|crs     |String|No|  coordinate reference system identifier for the **coords** values and output data (available options listed in the [collections response](collection_metadata_desc) response)
|f  |String|No| Data format for the output data (available options listed in the [collections response](collection_metadata_desc) response)

**Comment by chris-little** [comment]
@ghobona @cmheazel Are you happy with this approach? We can discuss and consider for version 1.0.1

**Comment by ghobona** [comment]
@chris-little Yes, that's fine. The changes will be treated as a corrigendum. Please send a brief summary of the changes to @ogcscotts once the draft corrigendum is ready for a TC vote.

**Comment by ghobona** [comment]
@chris-little The tables will need to be consistent with the Requirements, Abstract Tests, and OpenAPI definition document.

**Comment by dblodgett-usgs** [comment]
fixed by #390 -- to be reviewed there and subsequently.

**Comment by chris-little** [comment]
EDR API SWG 2022-09-22 Agreed to close as now incorporated into master Branch.
